### PR TITLE
feat: add support for Kubernetes 1.12.8 and 1.13.5 on Azure Stack

### LINKED ISSUE #1374: feat: pre-pull Azure Stack's custom hyperkube v1.12.8

### LINKED ISSUE #1375: feat: pre-pull Azure Stack's custom hyperkube v1.13.x

### PR DESCRIPTION
**Pull request by rjaini**
**Reason for Change**:

Adding 1.12.8 and 1.13.5 K8s images to be included in azurestack vhd.
Also pointing hyperkube and pause images to download from MCR.

**Issue Fixed**:
Fixes #1374 
Fixes #1375 

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by jadarsie** [comment]
lgtm

File: packer/install-dependencies.sh [hunk 1]
```diff
@@ -363,8 +363,11 @@ K8S_VERSIONS="
 1.14.1
 1.13.6
 1.13.5
+1.13.5-azs
 1.12.9
+1.12.9-azs
```
Comment (CecileRobertMichon) [inline packer/install-dependencies.sh]: see https://github.com/Azure/aks-engine/pull/1420, let's hold on 1.12.9

Comment (rjaini) [reply packer/install-dependencies.sh]: Sure.

File: docs/topics/azure-stack.md [hunk 2]
```diff
@@ -149,6 +149,17 @@ If you need to expose more than 5 services, then the recommendation is to route
 
 ## Frequently Asked Questions
 
+### Supported Kubernetes Versions
+
+These are the Kubernetes versions that you can deploy to Azure Stack using AKS Engine:
+
+- 1.13.5
+- 1.12.9
```
Comment (CecileRobertMichon) [inline docs/topics/azure-stack.md]: this also needs to be removed

Comment (rjaini) [reply docs/topics/azure-stack.md]: Removed.

**Comment by CecileRobertMichon** [comment]
The PR has a merge conflict @rjaini

**Comment by rjaini** [comment]
Update the PR with conflict resolved.

File: packer/install-dependencies.sh [hunk 3]
```diff
@@ -363,7 +363,10 @@ K8S_VERSIONS="
 1.14.1
 1.13.6
 1.13.5
+1.13.5-azs
+1.12.9
```
Comment (CecileRobertMichon) [inline packer/install-dependencies.sh]: this shouldn't add back 1.12.9... might be a rebase error?

Comment (rjaini) [reply packer/install-dependencies.sh]: Yup. Fixing it.

**Pull request merged**